src/app/shell/Header.tsx

```diff
@@ -112,6 +112,18 @@ export default function Header() {
 
   const installable = installPromptEvent || iosPwaAvailable;
 
+  const offerRelaunch =
+    // as an alternative to installing,
+    !isStandalone &&
+    !installable &&
+    // offer desktop users the choice to relaunch in a no-tabs, less-UI window
+    window.name !== 'dim-solo-window' &&
```

Comment:
I am a bit unsure why you want to lock someone down to doing this only once.

Potential use case:
1. Have dim open in a window with a billion tabs.
2. Open in a seperate window because reasons of PR
3. Original window gets too overwhelming so you just yolo close it
4. Start adding more tabs to window opened in 2
5. Open a billion tabs and go back to 1

In this case you wouldn't be able to split DIM out again.

Reply:
great point